cartaya/2/task2.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is my second task from Leo">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style.css">
+    <title>Task#2</title>
+</head>
+ <body>
+    <header>
+        <span id="h1">Intranet</span>

```

Comment:
Never use the attribute id for elements name, use class. the id is a global attribute that defines a unique identifier which must be unique in the whole document.